modules/release-notes/pages/relnotes.adoc

```diff
@@ -1,7 +1,7 @@
 = Release Notes for Couchbase Server 6.5
 
 Couchbase Server 6.5 brings many sought-after relational capabilities to the NoSQL world.
-This release seeks to improve developer agility, reduce total cost of ownership, enhance manageability, and provide more enterprise-grade security and performance. 
```

Comment:
lots of whitespace changes, may be worth turning this feature off in VSCode, to avoid making the diffs noisy.